visualization/tests/test_views.py

```diff
@@ -51,12 +51,20 @@ def setUpModule():
     CountryPartner.objects.create(
         name="united kingdom",
```

Comment:
name should be Kenya according to country_code_alpha

Reply:
done